zenodo/config.py

```diff
@@ -357,6 +357,54 @@ def _(x):
 GITHUB_PID_FETCHER = 'zenodo_doi_fetcher'
 #: GitHub metdata file
 GITHUB_METADATA_FILE = '.zenodo.json'
+#: GitHub error handlers
+GITHUB_ERROR_HANDLERS = [
+    (
```

Comment:
The ones we cannot handle can be left commented out (so that we know they exist), and then gradually we can start addressing them.